strawberry/types/schema/schema.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional, Set, Type
+
+from cached_property import cached_property
+
+from strawberry.backends import GraphQLCoreBackend, StrawberryBackend
+from strawberry.types import StrawberryObjectType, StrawberryType
+
+
+class StrawberrySchema:
+    def __init__(self, query: StrawberryObjectType, *,
+                 mutation: Optional[StrawberryObjectType] = None,
+                 subscription: Optional[StrawberryObjectType] = None,
+                 backend: Type[StrawberryBackend] = GraphQLCoreBackend):
+        self.backend = backend(query, mutation, subscription)
+
+    # TODO: I switched from execute_sync to execute_async as that appears to be
+    #       the standard for async "overloads". Certainly up for discussion;
+    #       leaning on switching it back
+    #       https://stackoverflow.com/q/52459138/8134178
```

Comment:
I kinda think that for graphql the default should async, since you get more benefits with it, also helps with pushing async adoption I guess ^^

Reply:
Sounds good enough to me :+1: